docs/ux-guidelines-payments/payment-button-size.md

```diff
@@ -0,0 +1,81 @@
+---
+post_title: User Experience Guidelines - Payment Button Size and Anatomy
+menu_title: Payment Button Size
+---
+
+Payment buttons must be consistent in visual appearance to other buttons in the shopper experience.
+
+Height for payment buttons ranges from 40px (Small) to 56px (Large). The default button height is 48px. The following presets can be used for button height.
+
+- Small: Height 40px
+- Default: Height 48px
+- Large: Height 56
+
+### Anatomy
+
+Payment buttons consist of up to three elements: a button component, payment logo, and optional label.
+[Anatomy.png]
+
+Don’t place any text outside the surface area of the button.
+
+![Cart layout 2](https://woo-docs-multi-com.go-vip.net/docs/wp-content/uploads/sites/3/2024/01/Cart-layout-2.png")
```

Comment:
@JacklynBiggin For some reason this image isn't loading. Let me know if you need me to resend.

Reply:
Good catch! Removed the " - should be fixed!